### DISCUSSION TITLE
Hope add only use hotkeys to wake screen

### DISCUSSION THREAD

**Discussion by sd2883809** [xanderfrangos/twinkle-tray#190] [Ideas]
My Laptop doesn't have turn off display function. This software is very helpful. I like it very much.

Unfortunately, Mouse and keyboard is very easy to be moved which can wake the screen.

Can you add a optional setting, only use turn off display hotkey to wake up screen?

Thank you.

**Comment by xanderfrangos** [comment]
Hi @sd2883809,

Unfortunately, it isn't possible for me to add that feature to Twinkle Tray. Laptops do not support turning off their displays through software like you could turn off a monitor.
